addon-test-support/index.js

```diff
@@ -0,0 +1,30 @@
+import { getContext } from '@ember/test-helpers';
+import { Promise } from 'rsvp';
+
+export async function waitForSource(sourceName) {
```

Comment:
I think it would be useful if `waitForSource` accepted `sourceOrSourceName` as an argument, so that the source would only be looked up by name if it wasn't directly provided.